CONTRIBUTING.txt

```diff
@@ -0,0 +1,24 @@
+Want to contribute? Great! First, read this page (including the small print at the end).
```

Comment:
Do we really want to include this file in versioning ?
I would rather have a dedicated tools subdirectory, and store all of them as a zip file